packages/react/src/components/Table/TableCellRenderer/TableCellRenderer.test.jsx

```diff
@@ -34,104 +38,81 @@ describe('TableCellRenderer', () => {
-        {cellText}
-      </TableCellRenderer>
-    );
-    expect(wrapper.find(`.${iotPrefix}--table__cell-text--no-wrap`)).toHaveLength(1);
+  it('allows wrapping by default', () => {
+    render(<TableCellRenderer>{cellText}</TableCellRenderer>);
+    expect(screen.getByText(cellText)).not.toHaveClass(textNoWrapClassName);
+    expect(screen.getByText(cellText)).not.toHaveClass(textTruncateClassName);
   });
 
-  it('allows wrap when wrapText={always}', () => {
-    const wrapper = mount(
-      <TableCellRenderer wrapText="always" truncateCellText={false}>
-        {cellText}
-      </TableCellRenderer>
-    );
-    expect(wrapper.find(`.${iotPrefix}--table__cell-text--no-wrap`)).toHaveLength(0);
+  it('allows wrapping for cellTextOverflow:"wrap" ', () => {
+    render(<TableCellRenderer cellTextOverflow="wrap">{cellText}</TableCellRenderer>);
+    // Wrapping is Carbon default, so the the absense of these classes
+    // indicates that the text is wrapping.
+    expect(screen.getByText(cellText)).not.toHaveClass(textNoWrapClassName);
+    expect(screen.getByText(cellText)).not.toHaveClass(textTruncateClassName);
   });
 
   it('only truncates children that are strings, numbers or booleans', () => {
-    const wrapper = mount(
+    render(
       <table>
         <tbody>
           <tr>
             <td>
-              <TableCellRenderer wrapText="never" truncateCellText>
-                {'my string'}
-              </TableCellRenderer>
+              <TableCellRenderer cellTextOverflow="truncate">my string</TableCellRenderer>
             </td>
             <td>
-              <TableCellRenderer wrapText="never" truncateCellText>
-                {true}
-              </TableCellRenderer>
+              <TableCellRenderer cellTextOverflow="truncate">{true}</TableCellRenderer>
             </td>
             <td>
-              <TableCellRenderer wrapText="never" truncateCellText>
-                {127}
-              </TableCellRenderer>
+              <TableCellRenderer cellTextOverflow="truncate">{127}</TableCellRenderer>
             </td>
             <td>
-              <TableCellRenderer wrapText="never" truncateCellText>
+              <TableCellRenderer cellTextOverflow="truncate">
                 <div>a div</div>
               </TableCellRenderer>
             </td>
           </tr>
         </tbody>
       </table>
     );
-    expect(wrapper.find(`.${iotPrefix}--table__cell-text--truncate`)).toHaveLength(3);
+    expect(screen.getByText('my string')).toHaveClass(textTruncateClassName);
+    expect(screen.getByText('true')).toHaveClass(textTruncateClassName);
+    expect(screen.getByText('127')).toHaveClass(textTruncateClassName);
+    expect(screen.getByText('a div').parentNode).not.toHaveClass(textTruncateClassName);
   });
 
   it('only shows tooltip if text is actually truncated', () => {
     setOffsetWidth(10);
     setScrollWidth(20);
 
     const wrapper = mount(
```

Comment:
switch these to testing-library, too? or are there a ton of tests in there and we'll do it later?

Reply:
Nope, just this old bad boy that somehow managed to survive. I'll rewrite it.

Reply:
Corrections, there are many. I'll fix them all.